src/main/resources/templates/account/emailUpdated.html

```diff
@@ -6,12 +6,19 @@
             <div class="grid-row">
                 <div class="govuk-box-highlight">
                     <h1 class="heading-xlarge">Change email complete</h1>
-                    <p class="font-medium">Your email has been successfully updated</p>
+                    <p class="font-medium">Your email updated email address
```

Comment:
Screen will need to be given the all clear from CSL.
Content Review part 3 required.

Reply:
Yeah, I was trying to follow content guidelines from https://design-system.service.gov.uk/components/panel/

I'm going to keep it in for this PR, but we can remove if needed in future.

Reply:
Yeah, just update Lisa.